src/Core/Managed/Shared/Extensibility/Implementation/TaskTimer.cs

```diff
@@ -17,15 +17,22 @@ namespace Microsoft.ApplicationInsights.Extensibility.Implementation
     /// <summary>
     /// Runs a task after a certain delay and log any error.
     /// </summary>
-    public class TaskTimer : IDisposable
+    public class TaskTimer
```

Comment:
Good point. `TaskTimer` is defined in the `Microsoft.ApplicationInsights` assembly because it is still used here as well as in other AI packages that depend on it. However, it doesn't need to be public anymore because we no longer publish WinRT assemblies. The class was implemented as public because WinRT didn't support internal class visibility at the time. Instead we placed it in the "Implementation" namespace and decided that semver does not apply to public types in this namespace. It should be safe to make `TaskTimer` internal straight away.